app/src/main/res/values-nb-rNO/strings.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+
+<!-- This file contains resource definitions for displayed strings, allowing
+     them to be changed based on the locale and options. -->
+
+<resources>
+    <!-- Simple strings. -->
+    <string name="mpv_activity">mpv</string>
+    <string name="title_activity_settings">Innstillinger</string>
+    <string name="dialog_ok">OK</string>
+    <string name="dialog_cancel">Avbryt</string>
+
+    <!--  Action buttons -->
+    <string name="action_settings">Innstillinger</string>
+    <string name="action_open_url">Åpne nettadrese</string>
+     
+    <!-- Strings related to Settings -->
+    <string name="pref_header_general">Generelt</string>
+
+    <string name="pref_default_file_manager_path_title">Sti til forvalgt filutforsker</string>
+    <string name="pref_default_file_manager_path_hint">Velg forvalgt filutforsker-sti</string>
+    <string name="pref_default_file_manager_path_summary">%s</string>
+
+    <string name="pref_default_audio_language_title">Forvalgt talespråk</string>
+    <string name="pref_default_audio_language_hint">Velg forvalgt talespråk for avspilling av video med mer enn ett språk</string>
+    <string name="pref_default_audio_language_summary">%s</string>
+
+    <string name="pref_default_subtitle_language_title">Forvalgt undertekst-språk</string>
+    <string name="pref_default_subtitle_language_hint">Velg forvalgt undertekst-språk for avspilling av video med mer enn ett språk</string>
+    <string name="pref_default_subtitle_language_summary">%s</string>
+
+    <string name="pref_hardware_decoding_title">Maskinvaredekoding</string>
+    <string name="pref_hardware_decoding_summary">Prøver maskinvarebasert dekoding, og faller tilbake på programvarebasert dekoding hvis det ikke virker</string>
+
+     <string name="pref_touch_gestures_title">Håndvendinger</string>
+     <string name="pref_touch_gestures_summary">Kontroller lys- og lyd-styrke, eller sanntidsspol ved bruk av håndvendinger.</string>
+
+    <string name="pref_header_video">Video</string>
+
+    <string name="pref_video_upscale_title">Oppskaleringsmetode</string>
+
+    <string name="pref_video_downscale_title">Nedskaleringsmetode</string>
+
+    <string name="pref_video_debanding_title">Fargeovergangsutgjevning</string>
+    <string name="pref_video_debanding_summary">Velg modus for utjevning av skilte fargeoverganger for video.</string>
+
+    <string name="pref_video_sync_title">Videosynkronisering</string>
+    <string name="pref_video_sync_summary">Velg hvordan avspilleren synkroniserer lyd og bilde</string>
+    <string name="pref_video_sync_default">audio</string> <!-- equal to mpv default, see options.rst -->
```

Comment:
entries without translated text should be removed from this file (the `%s`  and empty one too)

Reply:
Fixed.

Reply:
you forgot this one

Reply:
Forgot how?